blocks/highlight/highlight.js

```diff
@@ -1,5 +1,6 @@
 export default async function decorate(block) {
   let spotlight = document.createElement('div');
+  const hideSpotlight = (block.classList.contains('feed-newsroom'));
```

Comment:
How about just keeping the spotlight hidden through 
```
.block.feed-newsroom spotlight {
  display: none;
}
```

Reply:
feed-newsroom is a block which is not yet leveraged in this featured section. This PR is for the upper featured section which will be further as a fragment in newsroom page where to list all the cards - feed-newsroom block will be leveraged. Pl advise.

Reply:
If the spotlight is to be hidden, I think the authors should explicitly specify it as a block variant while authoring.